css-brochure-form/styles.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+  justify-content: center;
+}
+
+.form-container {
+  text-align: center;
+  margin: auto 200px;
+}
+
+h2 {
+  color: rgb(204, 90, 90);
+  text-transform: uppercase;
+  margin: 10px;
+  font-size: 12pt;
+  text-align: center;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  letter-spacing: 0.05rem;
+}
+
+.input {
+  height: 30px;
+  border-radius: 6px;
+  border-color: rgb(233, 227, 227);
+  border-style: solid;
+  width: 100%;
+  font-family: 'AtlasGrotesk', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+select {
+  background-color: rgb(229, 226, 226);
+}
+
+label {
+  font-size: 0.7rem;
+  text-transform: uppercase;
+  margin: 6px;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.email {
```

Comment:
This class and the three rules following it have the same styling. Rather than repeating this, you should either make one class for all of these elements, or make this a grouped selector for all of these classes on one CSS rule :grin: